### PR TITLE
:bug: handle empty params and empty inject function return

### LINKED ISSUE #81: Handle an empty parameter for inject
The inject function throws a cryptic error when no parameter is given :

```es6
import { inject } from 'k-ramel/react'
import Component from './Component'

export default inject()(Component)
```

image

### PR DESCRIPTION
**Pull request by EmrysMyrddin**
fixes #81 #82

### CODE REVIEW

File: packages/connectors/react/src/inject.jsx [hunk 1]
```diff
@@ -63,7 +63,9 @@ export default injectFunction => WrappedComponent => class extends Component {
   inject = (nextProps) => {
     this.setState(state => ({
       ...state,
-      injectedProps: injectFunction(this.context.store, nextProps || this.props),
+      injectedProps: injectFunction
+        ? injectFunction(this.context.store, nextProps || this.props) || {}
```
Comment (fabienjuif) [inline packages/connectors/react/src/inject.jsx]: default object should be a constant (outside the main function) so the shallowEquals is optimized

Comment (EmrysMyrddin) [reply packages/connectors/react/src/inject.jsx]: 👌

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (fabienjuif) [inline .vscode/settings.json]: Can you .gitignore that ?

Comment (EmrysMyrddin) [reply .vscode/settings.json]: It's already ignored in the PR #87. I delete it from this one.

**Comment by fabienjuif** [comment]
Thank you @EmrysMyrddin 😘

**Pull request closed without merging**